catalog/app/utils/yaml.ts

```diff
@@ -0,0 +1,18 @@
+import yaml from 'js-yaml'
+
+import * as Types from 'utils/types'
+
+export function stringify(inputObj?: Types.JsonRecord): string {
+  return yaml.dump(inputObj)
+}
+
+// eslint-disable-next-line consistent-return
+export function parse(inputStr?: string) {
+  if (!inputStr) return undefined
+  try {
+    return yaml.load(inputStr)
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error(error)
```

Comment:
shouldnt we re-throw the error here?

Reply:
You're right, we should. At least, for matching with `JSON.parse`
But current design doesn't expect this.

I'll refactor this later